### ISSUE TITLE
Support for Linux Xfce4

### ISSUE THREAD

**Issue by anfe0690** [AlfredoSequeida/venus#12]
Hi, support for the Linux desktop environment Xfce4 is missing. Can we add it?

**Comment by anfe0690** [comment]
I've created the pull request for this: #13

**Comment by AlfredoSequeida** [comment]
> Hi, support for the Linux desktop environment Xfce4 is missing. Can we add it?

Definitely, I'll look into it or if you want - you're welcome to try and add support for it and submit a pull request.

**Comment by anfe0690** [comment]
> Definitely, I'll look into it or if you want - you're welcome to try and add support for it and submit a pull request.

Please take a look at the pull request: #13

**Comment by AlfredoSequeida** [comment]
> > Definitely, I'll look into it or if you want - you're welcome to try and add support for it and submit a pull request.
> 
> Please take a look at the pull request: #13

Awesome, sorry for the wait, I'll take a look at it.

**Comment by AlfredoSequeida** [comment]
> > Definitely, I'll look into it or if you want - you're welcome to try and add support for it and submit a pull request.
> 
> Please take a look at the pull request: #13

So I tested it on Xfce 4.14 and it didn't work for me. The good thing is that I know why. 

Your current logic assumes that the property to set the wallpaper is that which contains "last-image" as part of the property. 

`if line.endswith("last-image")`

However, when I tried it, that was not the case for my system (tested using Linux mint 19.3). Monitoring Xfce for changes I noticed that the property used for setting the wallpaper for me was 

`/backdrop/screen0/monitoreDP-1/workspace0/last-image`

So, the logic needs to be updated so that it works with other configurations before It can be merged. The tricky part is figuring out how to distinguish between configurations so that Venus knows the property to set the image to since I suspect that it will be different based on versions of Xfce4 and monitor setups (currently only accounting for single monitor setups of course). I'll leave the logic up to you, but maybe looking at log files to see where the last image was set could work.
